dwitter/models.py

```diff
@@ -135,8 +135,7 @@ def __str__(self):
 # Should be idempotent.
 @receiver(post_save, sender=Comment, dispatch_uid="add_hashtags_from_comment")
 def add_hashtags(sender, instance, **kwargs):
-    hash_pattern = re.compile(r'#(?P<hashtag>[_a-zA-Z][_a-zA-Z\d]*)')
-    for hashtag in re.findall(hash_pattern, instance.text):
+    for hashtag in re.findall(r'(?<!\S)#([_a-zA-Z\d]+)\b)', instance.text):
```

Comment:
Did you intend to have a `)` both before and after the final word boundary here? Maybe the last one should go away?

```suggestion
    for hashtag in re.findall(r'(?<!\S)#([_a-zA-Z\d]+)\b', instance.text):
```

Reply:
Yeah... `')` not `)'`... Thanks :)